src/github.com/couchbase/sync_gateway/rest/changes_api.go

```diff
@@ -417,6 +423,7 @@ func (h *handler) sendContinuousChangesByWebSocket(inChannels base.Set, options
 
 		options.Terminator = make(chan bool)
 		defer close(options.Terminator)
+		defer h.db.DatabaseContext.TapListener.Notify(base.SetOf("user"))
```

Comment:
I'd probably prefer calling a new method on DatabaseContext to do the notification (like NotifyUser()), instead of changing the scope of tapListener.  That would also allow reuse for future use cases, and could encapsulate the calculation of the user key.

User key is the user doc name.  It gets added as a wait key here:
https://github.com/couchbase/sync_gateway/blob/master/src/github.com/couchbase/sync_gateway/db/change_listener.go#L164
and a sample notify usage is here: https://github.com/couchbase/sync_gateway/blob/master/src/github.com/couchbase/sync_gateway/db/change_listener.go#L57